docs/python-support.md

```diff
@@ -1,23 +1,20 @@
-# Python Support in Djinni
+# Python Support
 
-Python support in Djinni is experimental, but ready to try out.  It can generate code for bridging C++ with Python 3.
+Python support in Djinni is experimental, but ready to try out.  It can generate code for bridging
+C++ with Python 3.
 
-For more information, you can check out our talk from CppCon 2015.  [Slides](https://bit.ly/djinnitalk2) and [video](https://bit.ly/djinnivideo2) are avilable online.
-
-## Generated Code
-
-When bridging to Python, Djinni generates 4 types of output:
-
-* `python` Generated Python classes and proxies for interacting with C++ via [CFFI](https://cffi.readthedocs.org/).
-* `cffi` Python code run at build time to create a Python extension out of the C++ code.
-* `cwrapper` A C interface implemented in C++ to allowing Python to interact with C++ classes.
-* `cpp` The same C++ classes generated for all other Djinni languages.
+For more information, you can check out the talk from CppCon 2015.
+[Slides](https://bit.ly/djinnitalk2) and [video](https://bit.ly/djinnivideo2) are avilable online.
 
 ## Pre-requisites
 
-Djinni generates code which should work with Python 3.  Python support has only been tested on macOS with the clang compiler.  ~~It's been designed to work on any platform, and the known issues are in the build scripts rather than the generated code itself.~~ _(We're now using CMake to build all bindings but it still requires testing in platforms other than macOS.)_
+Djinni generates code which should work with Python 3.  Python support has only been tested on macOS
```

Comment:
As mentioned [here](https://github.com/cross-language-cpp/djinni-support-lib/pull/24#issuecomment-819568338) I think a matrix build & test run could give us more confidence in platform compatibility. I'm pretty invested in C# support right now but I'll try to make a proposal for the github action next week.

Reply:
Not gonna happen in this PR. But it may be still worth investing some time into this in the future :)

Reply:
Not gonna happen in this PR. But it may be still worth investing some time into this in the future :)